collectors/perf.plugin/perf_plugin.c:171: error: 'PERF_COUNT_HW_REF_CPU_CYCLES' undeclared here (not in a function)

<!---
This is a generic issue template. We usually prefer contributors to use one
of 3 other specific issue templates (bug report, feature request, question)
to allow our automation classify those so you can get response faster.
However if your issue doesn't fall into either one of those 3 categories
use this generic template.
--->

#### Summary
collectors/perf.plugin/perf_plugin.c:171: error: 'PERF_COUNT_HW_REF_CPU_CYCLES' undeclared here (not in a function)
daemon/global_statistics.c:59:6: warning: #warning NOT using atomic operations - using locks for global statistics
daemon/global_statistics.c:89:2: warning: #warning NOT using atomic operations - using locks for global statistics
make64[2]: *** [collectors/perf.plugin/perf_plugin.o] Error 1


met this error when install netdata

--- a/configure.ac
+++ b/configure.ac
@@ -737,8 +737,19 @@ AM_CONDITIONAL([ENABLE_PLUGIN_XENSTAT], [test "${enable_plugin_xenstat}" = "yes"
 # -----------------------------------------------------------------------------
 # perf.plugin
 
+AC_CHECK_HEADER(
+    [linux/perf_event.h],
+    [AC_CHECK_DECL(
+        [PERF_COUNT_HW_REF_CPU_CYCLES],
+        [have_perf_event=yes],
+        [have_perf_event=no],
+        [#include <linux/perf_event.h>]
+    )],
+    [have_perf_event=no]
+)
+
 AC_MSG_CHECKING([if perf.plugin should be enabled])
-if test "${build_target}" == "linux" ; then
+if test "${build_target}" == "linux" -a "${have_perf_event}" = "yes"; then
     enable_plugin_perf="yes"
 else
     enable_plugin_perf="no"

